source/Tests/Playnite.Tests/SearchViewModelTests.cs

```diff
@@ -15,30 +15,44 @@ public class SearchViewModelTests
         [SetCulture("en-US")]
         public void MatchTextFilterTest()
         {
-            Assert.IsTrue(SearchViewModel.MatchTextFilter("c", "has chalupa"));
-            Assert.IsTrue(SearchViewModel.MatchTextFilter("op stea", "open steam settings"));
-            Assert.IsTrue(SearchViewModel.MatchTextFilter("op STea", "open settings steam"));
-            Assert.IsTrue(SearchViewModel.MatchTextFilter("open steam", "open steam settings"));
-            Assert.IsTrue(SearchViewModel.MatchTextFilter("", "open steam settings"));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("c", "has chalupa", false));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("op stea", "open steam settings", false));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("op STea", "open settings steam", false));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("open steam", "open steam settings", false));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("", "open steam settings", false));
 
-            Assert.IsFalse(SearchViewModel.MatchTextFilter("opass stea", "open steam settings"));
-            Assert.IsFalse(SearchViewModel.MatchTextFilter("test", ""));
+            Assert.IsFalse(SearchViewModel.MatchTextFilter("opass stea", "open steam settings", false));
+            Assert.IsFalse(SearchViewModel.MatchTextFilter("test", "", false));
 
-            Assert.IsTrue(SearchViewModel.MatchTextFilter("", ""));
-            Assert.IsTrue(SearchViewModel.MatchTextFilter(null, null));
-            Assert.IsTrue(SearchViewModel.MatchTextFilter("", null));
-            Assert.IsTrue(SearchViewModel.MatchTextFilter(null, ""));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("", "", false));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter(null, null, false));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("", null, false));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter(null, "", false));
+
+            // Acronym tests
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("gow", "Gears of War", true));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("goW", " Gears of War", true));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("GOW", "god of war 3", true));
+            Assert.IsFalse(SearchViewModel.MatchTextFilter(" GOW", "god of war 3", true));
+            Assert.IsFalse(SearchViewModel.MatchTextFilter("GOW ", "god of war 3", true));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("GOW", "god   of war   3", true));
+            Assert.IsFalse(SearchViewModel.MatchTextFilter("g OW", "god of war 3", true));
+            Assert.IsTrue(SearchViewModel.MatchTextFilter("gOw3", "god of war 3", true));
+            Assert.IsFalse(SearchViewModel.MatchTextFilter("gOw3", "god of war 2", true));

```

Comment:
Missing tests with disabled acronym support I mentioned in my original comment.

Reply:
I think I did what you indicate, please let me know if it's okay